pkgs/tools/misc/remote-exec/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, fetchFromGitHub
+, buildPythonApplication
+# runtime
+, click
+, toml
+, pydantic
+, watchdog
+# check
+, pytestCheckHook
+, rsync
+}:
+
+buildPythonApplication rec {
+  pname = "remote-exec";
+  version = "1.13.2";
+
+  src = fetchFromGitHub {
+    owner = "remote-cli";
+    repo  = "remote";
+    rev   = "refs/tags/v${version}";
+    hash  = "sha256-xaxkN6XukV9HiLYehwVTBZB8bUyjgpfg+pPfAGrOkgo=";
+  };
+
+  # remove legacy endpoints, we use --multi now
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace '"mremote' '#"mremote'
+  '';
+
+  propagatedBuildInputs = [
+    click
+    toml
+    pydantic
+    watchdog
+  ];
+
+  # We don't need pytest --cov
+  preCheck = ''
+    rm setup.cfg
+  '';
+
+  doCheck = true;
+  checkInputs = [
+    pytestCheckHook
+  ];
+  nativeBuildInputs = [
+    rsync # for some reason it doesn't work when put in checkInputs
+  ];
+
+  meta = with lib; {
+    description = "Work with remote hosts seamlessly via rsync and ssh. Code locally, build and execute remotely";
+    homepage = "https://github.com/remote-cli/remote";
+    platform = platforms.all;
```

Comment:
```suggestion
```

The correct name is `platforms`.
Also we don't set platforms on Python packages (except for restricting them).